### PR TITLE
remove sig machinery from sig-mention-handler munger

### PR DESCRIPTION
**Pull request by crimsonfaith91**
addressing https://github.com/kubernetes/test-infra/issues/3494

### CODE REVIEW

File: mungegithub/mungers/sig-mention-handler.go [hunk 1]
```diff
@@ -107,7 +107,7 @@ func (s *SigMentionHandler) Munge(obj *github.MungeObject) {
 There are no sig labels on this issue. Please [add a sig label](https://github.com/kubernetes/test-infra/blob/master/commands.md) by:
 	
 1. mentioning a sig: `+"`@kubernetes/sig-<group-name>-<group-suffix>`"+`
-    e.g., `+"`@kubernetes/sig-api-machinery-<group-suffix>`"+` to notify the API Machinery sig, OR
+    e.g., `+"`@kubernetes/contributor-experience-<group-suffix>`"+` to notify the contributor experience sig, OR
```
Comment (cblecker) [inline mungegithub/mungers/sig-mention-handler.go]: I think the `sig-` part was left out from `@kubernetes/contributor-experience-<group-suffix>`

**Comment by crimsonfaith91** [comment]
@cblecker Yes! I pointed out in this issue too: https://github.com/kubernetes/kubernetes/issues/48916

**Pull request merged**